.vscode/tasks.json

```diff
@@ -0,0 +1,37 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "build and run",
+            "type": "shell",
+            "command": "make; ./rpi-imager",
```

Comment:
Will this still try to run `./rpi-imager` (e.g. a previously-built version) if `make` fails? Might be better to use `make && ./rpi-imager` ?